pkg/runtime/grpc/grpc-runtime.go

```diff
@@ -387,7 +440,8 @@ func (r *Runtime) runGadget(gadgetCtx runtime.GadgetContext, target target, allP
 	connCtx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	dialCtx, cancelDial := context.WithTimeout(gadgetCtx.Context(), time.Second*ConnectTimeout)
+	timeout := time.Second * time.Duration(r.globalParams.Get(ParamConnectionTimeout).AsInt())
```

Comment:
Can this variable be declared once before L411 and used in both places? (L415 and L444)

Reply:
Hmm, L444 is another function - it could be defined in `runGadget` and then used as parameter to `dialContext` to be reused, but then the other callers of `dialContext` would have to get the timeout value. I'll change it to that.